examples/repl/cmdline.rs

```diff
@@ -25,65 +25,58 @@ use num_traits::FromPrimitive;
 pub unsafe fn dc_reset_tables(context: &Context, bits: i32) -> i32 {
     info!(context, 0, "Resetting tables ({})...", bits);
     if 0 != bits & 1 {
-        dc_sqlite3_execute(
-            context,
-            &context.sql,
-            b"DELETE FROM jobs;\x00" as *const u8 as *const libc::c_char,
-        );
+        dc_sqlite3_execute(context, &context.sql, "DELETE FROM jobs;", params![]);
         info!(context, 0, "(1) Jobs reset.");
     }
     if 0 != bits & 2 {
         dc_sqlite3_execute(
             context,
             &context.sql,
-            b"DELETE FROM acpeerstates;\x00" as *const u8 as *const libc::c_char,
+            "DELETE FROM acpeerstates;",
+            params![],
         );
         info!(context, 0, "(2) Peerstates reset.");
     }
     if 0 != bits & 4 {
-        dc_sqlite3_execute(
-            context,
-            &context.sql,
-            b"DELETE FROM keypairs;\x00" as *const u8 as *const libc::c_char,
-        );
+        dc_sqlite3_execute(context, &context.sql, "DELETE FROM keypairs;", params![]);
         info!(context, 0, "(4) Private keypairs reset.");
     }
     if 0 != bits & 8 {
         dc_sqlite3_execute(
             context,
             &context.sql,
-            b"DELETE FROM contacts WHERE id>9;\x00" as *const u8 as *const libc::c_char,
+            "DELETE FROM contacts WHERE id>9;",
+            params![],
         );
         dc_sqlite3_execute(
             context,
             &context.sql,
-            b"DELETE FROM chats WHERE id>9;\x00" as *const u8 as *const libc::c_char,
+            "DELETE FROM chats WHERE id>9;",
+            params![],
         );
         dc_sqlite3_execute(
             context,
             &context.sql,
-            b"DELETE FROM chats_contacts;\x00" as *const u8 as *const libc::c_char,
+            "DELETE FROM chats_contacts;",
+            params![],
         );
         dc_sqlite3_execute(
             context,
             &context.sql,
-            b"DELETE FROM msgs WHERE id>9;\x00" as *const u8 as *const libc::c_char,
+            "DELETE FROM msgs WHERE id>9;",
+            params![],
         );
         dc_sqlite3_execute(
             context,
             &context.sql,
-            b"DELETE FROM config WHERE keyname LIKE \'imap.%\' OR keyname LIKE \'configured%\';\x00"
-                as *const u8 as *const libc::c_char,
-        );
-        dc_sqlite3_execute(
-            context,
-            &context.sql,
-            b"DELETE FROM leftgrps;\x00" as *const u8 as *const libc::c_char,
+            "DELETE FROM config WHERE keyname LIKE \'imap.%\' OR keyname LIKE \'configured%\';",
```

Comment:
is this escaping of `'` neccewsary here?

Reply:
I am not sure, I don't touch those sql strings unless I have to

Reply:
the escaping is not needed by sqlite here, in fact, sqlite must get a literal `'` without backslash.

so the escaping here is only for the rust-syntax, if i looks at the examples at https://doc.rust-lang.org/rust-by-example/std/str.html , escaping `'` in strings delimited by `"` seems not to be needed, however, not sure :)

Reply:
removed it was not needed indeed

Reply:
not repushed?

Reply:
forgot that file